charmed_openstack_info/data/lp-builder-config/openstack.yaml

```diff
@@ -1657,7 +1612,7 @@ projects:
         build-channels:
           charmcraft: "2.x/stable"
         channels:
-          - yoga/candidate
+          - yoga/stable
```

Comment:
no, we want to push to candidate until we get positive confirmation that this charm is fine for yoga.

Reply:
We got confirmation from a deployment using yoga/candidate and all went OK, so we can promote this stable now.